etc/schema.graphqls

```diff
@@ -5,7 +5,7 @@ type StoreConfig @doc(description: "The type contains information about a store
 
 type Mutation {
 	setRzpPaymentDetailsForOrder(input: SetRzpPaymentDetailsForOrderInput) : SetRzpPaymentDetailsForOrderOutput @resolver(class: "Razorpay\\Magento\\Model\\Resolver\\SetRzpPaymentDetailsForOrder")
-	placeRazorpayOrder (order_id: String @doc(description: "order_id to generate Razorpay Order ID.")) : RazorpayOrder @resolver( class: "Razorpay\\Magento\\Model\\Resolver\\PlaceRazorpayOrder") @doc(description: "Place Razorpay Order with cart amount and currency to generate RZP order ID.")
+	placeRazorpayOrder (input: PlaceRazorpayOrderInput) : RazorpayOrder @resolver( class: "Razorpay\\Magento\\Model\\Resolver\\PlaceRazorpayOrder") @doc(description: "Place Razorpay Order with cart amount and currency to generate RZP order ID.")
```

Comment:
@yashgit891 , can we pass 2 input params directly here, in place of using it as input array?